Framework/CurveFitting/src/Algorithms/Fit.cpp

```diff
@@ -48,6 +48,9 @@ void Fit::initConcrete() {
       ->setDocumentation("Whether the fit was successful");
   declareProperty("OutputChi2overDoF", 0.0, "Returns the goodness of the fit",
                   Kernel::Direction::Output);
+  declareProperty("OutputNIterations", 0, Kernel::Direction::Output);
```

Comment:
Python returns a `namedtuple` now, there are no indices any more.

There is even more interesting case of dynamically created output properties by `Fit`: `FABADAMinimizer` defines its own outputs (probability distributions, etc)